### ISSUE TITLE
data_test.npz

### ISSUE THREAD

**Issue by IbrahimOjeili** [hamuchiwa/AutoRCCar#121]
Hello,

I am using data_test.npz to train the network using model_training, but I am getting 36% train accuracy and 13% validation accuracy. Is this the same data that you achieved 96% on? If not then is this data not suitable to train the NN?

**Comment by IbrahimOjeili** [comment]
Hello, 

I see, can you please post the training data you used for your car? Thank you!

**Comment by hamuchiwa** [comment]
My training data may not be useful to you, you need to collect your own :)
